lib/resolver.manifest.js

```diff
@@ -56,15 +59,20 @@ module.exports = class PodletClientManifestResolver {
                 headers,
             };
 
-            const timer = this.metrics.timer({
-                name: 'podlet_manifest_request',
+            const histogram = this.metrics.histogram({
+                name: 'podium_client_resolver_manifest_resolve',
                 description:
-                    'Time taken for success/failure of mainfest request',
-                meta: {
+                    'Time taken for success/failure of manifest request',
+                labels: {
                     url: reqOptions.uri,
```

Comment:
We can use `outgoing.manifestUri` instead here.

Reply:
came up undefined

Reply:
scratch that, works fine